lib/src/fnc/type.rs

```diff
@@ -91,7 +54,7 @@ pub fn thing((arg1, arg2): (Value, Option<Value>)) -> Result<Value, Error> {
 				Value::Thing(v) => v.id,
 				Value::Array(v) => v.into(),
 				Value::Object(v) => v.into(),
-				Value::Number(Number::Int(v)) => v.into(),
+				Value::Number(v) => v.into(),
```

Comment:
I haven't reached the part where `impl From<Number> for Id` is defined/redefined but it seems _suspicious_ to infallibly convert a float/decimal to an `Id`.